server/routes/signup.js

```diff
@@ -0,0 +1,25 @@
+let express = require('express');
```

Comment:
are you supposed to handle all of auth in this PR? In that case the signin route is missing